peru/parser.py

```diff
@@ -103,7 +103,15 @@ def _extract_modules(blob, name_prefix):
 
 
 def _build_module(name, type, blob, yaml_name):
-    peru_file = typesafe_pop(blob, 'peru file', DEFAULT_PERU_FILE_NAME)
+    peru_filename = DEFAULT_PERU_FILENAMES[0]
+    for filename in DEFAULT_PERU_FILENAMES:
+        try:
+            with open(filename):
```

Comment:
It looks like you're checking to see whether files of a given name exist in the current directory. That's not going to be what you want, for a couple big reasons:

- Like git, you can run peru in a subdirectory of a project, and it'll walk up the directory tree to find the root and do the right thing. However, we never actually `chdir` in the peru process. (That would be changing global state, which gets buggy and confusing really fast.) Instead we build all the absolute paths we need inside of `runtime.py`, and join things to those. Take a look at what `find_project_file` in `runtime.py` is doing, for example.
- The `peru file` field here refers to a filepath in a [recursive module](https://github.com/buildinspace/peru#recursion). It's not actually a file on the current machine (yet). You'll need to follow this value through the code in `module.py` to see how it gets used.